src/importlinter/contracts/_common.py

```diff
@@ -137,20 +139,43 @@ def _pop_shortest_chains(graph: ImportGraph, importer: str, imported: str):
             yield chain
 
 
-def format_line_numbers(line_numbers: Sequence[Optional[int]]) -> str:
+def format_line_numbers(format: str, line_numbers: Sequence[Optional[int]]) -> str:
```

Comment:
'format' feels much too broad a name. 'importer_format' maybe?

I'm not sure the best way to model this as it feels very customised to your specific needs - does displaying like this (with filename:line_number) have any established name?

It makes me wonder if it would be better to use some kind of templating / rendering hook that allows users complete control of the rendering of the contracts.